src/main/java/com/adomasda/Main.java

```diff
@@ -2,6 +2,14 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.err.println("TODO!");
+        Floor floor1 = new Floor(new Configuration(1, "available", SomeEnum.FIRST_FLOOR));
+        Floor floor2 = new Floor(new Configuration(2, "available", SomeEnum.SECOND_FLOOR));
+        Floor floor3 = new Floor(new Configuration(3, "unavailable", SomeEnum.SECOND_FLOOR));
+        Floor floor4 = new Floor(new Configuration(4, "available", SomeEnum.SECOND_FLOOR));
```

Comment:
what is the purpose of this configuration if everything is setup twice - floors numbers, availability.....
In case of change configuration the code change will be required.